examples/browser/webpack.config.js

```diff
@@ -52,6 +53,10 @@ module.exports = {
                 loader: 'source-map-loader'
             },
             {
+                test: /node_modules.+xterm.+\.map$/,
+                loader: 'ignore-loader'
+            },
```

Comment:
What is that ? ignore-loader for xterm.map ??

Reply:
This is to take care of the following lines when running webpack, it removes it:
You may need an appropriate loader to handle this file type.
| {"version":3,"sources":["../../../src/addons/search/SearchHelper.ts"],"names":[],"mappings":";;AAgBA;IACE,sBAAoB,SAAc,EAAU,4BAAiC;QAAzD,cAAS,GAAT,SAAS,CAAK;QAAU,iCAA4B,GAA5B,4BAA4B,CAAK;IAK7E,CAAC;IAQM,+BAAQ,GAAf,UAAgB,IAAY;QAC1B,EAAE,CAAC,CAAC,.....

Reply:
Remove from this request, , fill with issue #538